Extension/src/LanguageServer/configurations.ts

```diff
@@ -102,7 +102,7 @@ export interface ConfigurationErrors {
 }
 
 export interface Browse {
-    path?: string[];
+    path?: string[] | null;
```

Comment:
This isn't needed if the other changes I suggested are made.

Reply:
Actually, I changed my mind -- browse.path defaults to null, so this is fine.

Reply:
If we don't need the null value to signify something important, `undefined` should be used.